feat(content): Add three more human starts (#11302)
endless-sky/endless-sky: data/human/deep missions.txt

			`	"Let me answer that question with a short story of my life." Pierre clears his throat and puffs up his chest a little, as if preparing himself to tell a great epic. He may enjoy talking about himself a little too much...`
			`	"I was born on Windblain in the Regor system. It's a small, quiet world of little galactic importance, but you should give it a visit sometime. My family wasn't the most well off, so I needed to work hard in my studies, which paid off immensely. Thanks to my exceptional grades in university, I was offered a government job even before graduation. I've been working directly for the Deep for the past 35 years and counting, and I've loved every minute of it! My younger sister though... well, let's just say she wasn't so focused on being successful in school and ended up on the wrong side of the law.`
			`	"How about you?"`

> 

Reviewer: ```suggestion
```

			`	You tell Pierre about your life on <starting planet>, and how you saved up your money working at the textile mill from the age of fifteen until you were able to buy a pilot's license and finally leave your home world for the stars.`
				to display
					has "start: default"
			`	You tell Pierre about your life on <starting planet>, and how you graduated with high enough grades to apply for the Deep's starship loan program.`
				to display
					has "start: deep"
			`	You tell Pierre about your life on <starting planet>, and how you've always dreamed of becoming a starship captain, and finally got your wish when you were given a pilot's license on your eighteenth birthday.`
				to display
					has "start: paradise"

> 			`	You tell Pierre about your life moving around the galaxy, and how you worked on a Syndicate factory world for five years before you took a loan and begain your life as a merchant captain.`

Author: `<starting planet>` aliases to the player's homeworld (Earth, in this situation).

				to display
					has "start: syndicate"
			`	Making sure to not bring up <starting planet> or the Hai by name, you tell Pierre about your life, and how you did your best at a travel agency to get to the stars as soon as you could.`
				to display
					has "start: hai"

			`	"We aren't too different then," Pierre remarks. "We both worked hard to get where we are today. Thanks for sharing, <first>."`
				to display
					not "start: deep"
					not "start: paradise"
			`	"So you're native to the Deep," Pierre remarks. "I felt like an outsider when I first arrived. Compared to how welcoming people in the Dirt Belt are, it seemed like nobody wanted to talk to me. I've gotten over that since, of course. Thanks for sharing, <first>."`
				to display
					has "start: deep"
			`	"So you're from the Paradise Worlds," Pierre remarks. "Most people from there choose to settle down and take up their parents' businesses. Very few go out on their own as a merchant. Thanks for sharing, <first>."`
				to display
					has "start: paradise"

> 

Reviewer: ```suggestion
```

			`	"Let me answer that question with a short story of my life." Pierre clears his throat and puffs up his chest a little, as if preparing himself to tell a great epic. He may enjoy talking about himself a little too much...`
			`	"I was born on Windblain in the Regor system. It's a small, quiet world of little galactic importance, but you should give it a visit sometime. My family wasn't the most well off, so I needed to work hard in my studies, which paid off immensely. Thanks to my exceptional grades in university, I was offered a government job even before graduation. I've been working directly for the Deep for the past 35 years and counting, and I've loved every minute of it! My younger sister though... well, let's just say she wasn't so focused on being successful in school and ended up on the wrong side of the law.`
			`	"How about you?"`

> 			`	You tell Pierre about your life on <starting planet>, and how you saved up your money working at the textile mill from the age of fifteen until you were able to buy a pilot's license and finally leave your home world for the stars.`

Reviewer 2: ```suggestion
			`	You tell Pierre about your life on <home planet>, and how you saved up your money working at the textile mill from the age of fifteen until you were able to buy a pilot's license and finally leave your home world for the stars.`
```

				to display
					has "start: default"
			`	You tell Pierre about your life on <starting planet>, and how you graduated with high enough grades to apply for the Deep's starship loan program.`
				to display
					has "start: deep"

> 			`	You tell Pierre about your life on <starting planet>, and how you've always dreamed of becoming a starship captain, and finally got your wish when you were given a pilot's license on your eighteenth birthday.`

Reviewer 2: If the other location gets changed from "gave" to "bought" then perhaps this one should too.

				to display
					has "start: paradise"
			`	You tell Pierre about your life moving around the galaxy, and how you worked on a Syndicate factory world for five years before you took a loan and began your life as a merchant captain.`
				to display
					has "start: syndicate"
			`	Making sure to not bring up <starting planet> or the Hai by name, you tell Pierre about your life, and how you did your best at a travel agency to get to the stars as soon as you could.`
				to display
					has "start: hai"

			`	"We aren't too different then," Pierre remarks. "We both worked hard to get where we are today. Thanks for sharing, <first>."`
				to display
					not "start: deep"
					not "start: paradise"

> 			`	"So you're native to the Deep," Pierre remarks. "I felt like an outsider when I first arrived. Compared to how welcoming people in the Dirt Belt are, it seemed like nobody wanted to talk to me. I've gotten over that since, of course. Thanks for sharing, <first>."`

Reviewer 3: [Can confirm](https://github.com/endless-sky/endless-sky/blob/34fa67085b1f395b29f4f420125480f99840c5e4/data/human/deep%20missions.txt#L1247). This phrasing that he felt like an outsider initially jars with his earlier characterization where he's quite proud and "loved every minute" of his time studying and working for the Deep. 

How about tweaking to related to Pierre's pride in the Deep, his laser research background, and his enjoyment of talking about himself?

```suggestion
			`	"So you're from the Deep too," Pierre grins. "That loan program's tough - well done. Back in my university days, we helped build some of the subsystems for those loaned ships in an applied engineering class. You have to admire the Deep's meritocracy. Thanks for sharing, <first>."`
```

				to display
					has "start: deep"

> 			`	"So you're from the Paradise Worlds," Pierre remarks. "Most people from there choose to settle down and take up their parents' businesses. Very few go out on their own as a merchant. Thanks for sharing, <first>."`

Reviewer 3: ```suggestion
			`	"So you're from the Paradise Worlds," Pierre remarks. "Most people from there settle down to take up their parents' businesses. Very few go out on their own as a merchant. Thanks for sharing, <first>."`
```

Reviewer 2: I think this is fine the way it is.
